google-cloud-pubsub/samples/acceptance/topics_test.rb

```diff
@@ -40,15 +41,18 @@
   let(:role) { "roles/pubsub.publisher" }
   let(:service_account_email) { "serviceAccount:kokoro@#{pubsub.project}.iam.gserviceaccount.com" }
   let(:topic_id) { random_topic_id }
+  let(:cloud_storage_ingestion_topic_id) { random_topic_id }
+  let(:cloud_storage_bucket) { "pubsub-ruby-sample-bucket" }
   let(:subscription_id) { random_subscription_id }
   let(:dead_letter_topic_id) { random_topic_id }
 
   after do
     @subscription.delete if @subscription
     @topic.delete if @topic
+    @cloud_storage_ingestion_topic.delete if @cloud_storage_ingestion_topic
```

Comment:
I don't see where this instance variable gets set. Did you intend to set it in the test below (i.e. instead of just setting a local variable in line 74)?